arkouda/dataframe.py

```diff
@@ -2111,6 +2152,33 @@ def coargsort(self, keys, ascending=True):
             i = i[arange(self.size - 1, -1, -1)]
         return i
 
+    def __reindex(self, idx):
```

Comment:
Does this need to have a double underscore or just one? I know there are a few reserved functions that are reserved for it.

Reply:
I think we tend to use only one underscore for our own hidden functions to more easily differentiate between them and the [dunder functions](https://www.geeksforgeeks.org/dunder-magic-methods-python/) (double underscore). This is ultimately a stylistic choice but for the sake of a unified style, I'd be in favor of changing this to follow our convention of a single leader underscore

Reply:
OK, I changed it.  Thanks!